tag_to_header.py

```diff
@@ -243,8 +243,8 @@ def main():
 					dcs_count += 1
 
 					if o.reduce and tag not in dcs_tags_list:
```

Comment:
even if I'm using a set I left `tag not in` because I don't know if `tag[o.taglen:] + tag[:o.taglen]` could differ w/ the same `tag` ...